src/shared/sol-util-file.c

```diff
@@ -26,6 +26,7 @@
 #include <fcntl.h>
 #include <libgen.h>
 #include <limits.h>
+#include <sched.h>
```

Comment:
On Wed, 06 Apr 2016 07:23:22 -0700, Gustavo Lima Chaves wrote:
> > @@ -26,6 +26,7 @@
> >  #include <fcntl.h>
> >  #include <libgen.h>
> >  #include <limits.h>
> > +#include <sched.h>
> 
> ifdef under linux as well?

Or remove the whole thing? Why do we need to yield()? Most likely that
function will be used from the main thread anyway.